applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -212,15 +211,30 @@ def Initialize(self):
 
         # Initialize the distance correction process
         self._GetDistanceModificationProcess().ExecuteInitialize()
-
+        self._GetDistanceModificationProcess().ExecuteInitializeSolutionStep()
```

Comment:
We require this in here in order to prevent zero distance cuts when calculating the initial volume right?